packages/cache/src/main.ts

```diff
@@ -1,5 +1,5 @@
 export { caches } from './polyfill.js'
-export { setCacheHeaders } from './cache-headers/cache-headers.js'
+export { cacheHeaders, setCacheHeaders } from './cache-headers/cache-headers.js'
```

Comment:
perhaps we should rename the method to `netlifyCacheHeaders` as it is not a generic cache headers function, it generates the netlify-cdn-cache-control, netlify-vary, netlify-cache-tag, and netlify-cache-id headers - what do you think?